templates/app/models/index.js

```diff
@@ -19,7 +19,11 @@ let databaseOptions = {
 };
 
 if (process.env.DATABASE_SSL && JSON.parse(process.env.DATABASE_SSL.toLowerCase())) {
-  databaseOptions.dialectOptions.ssl = true;
+  if (process.env.DATABASE_URL.startsWith('mysql://')) {
+    databaseOptions.dialectOptions.ssl = { rejectUnauthorized: true };
+  } else {
+    databaseOptions.dialectOptions.ssl = true;
+  }
```

Comment:
I am wondering if this is a good idea or not to insert conditions like `if mysql` in a generated server that will never use `mysql` databases.
Something like:
```javascript
<% if (config.dbDialect === 'mysql') {
%>databaseOptions.dialectOptions.ssl = { rejectUnauthorized: true };
<% } else {
%>databaseOptions.dialectOptions.ssl = true;
<% } %>
```